pylintrc.tests

```diff
@@ -0,0 +1,7 @@
+[entrypoint-tests]
```

Comment:
is it possible to specify this on the pytest module rather than the importing entry point test?

Reply:
I didn't rename this title from #25, I changed it to `tests-configuration`.

Unsure about what you're asking for, what  does "on the pytest module" mean and what do you mean by importing entrypoint test in this case ? You think about moving the pylintrc to the tests' location, doing something from pytest ?

Or configure the pylintrc per directory ? This is not possible for now, they are doing an entire development to enable configuration per directory (see [project roadmap](https://github.com/PyCQA/pylint/projects/1) and issue *"[Add different configuration for different subdirectories](https://github.com/PyCQA/pylint/issues/618)"*.

Reply:
I'm looking for a way to ignore pytest and not our own code. Perhaps it's easier to not import pytest for lining and annotate the import with an ignore? I was mistakenly under the impression that the rc file was specifying these ignores for the entry point test module - due to the header.

Reply:
For now I filed #53 to unblock this.